src/vcpkg/postbuildlint.cpp

```diff
@@ -431,22 +405,20 @@ namespace vcpkg::PostBuildLint
             }
             else
             {
-                Checks::exit_with_message(VCPKG_LINE_INFO,
-                                          "Running command:\n   %s\n failed:\n%s\n",
-                                          cmd_line.command_line(),
-                                          maybe_output.error());
+                Checks::msg_exit_with_message(
+                    VCPKG_LINE_INFO,
+                    msg::format(msgRunningCommandFailed, msg::command_line = cmd_line.command_line())
+                        .append_raw('\n')
+                        .append(maybe_output.error()));
             }
         }
 
         if (!dlls_with_no_exports.empty())
         {
-            print2(Color::warning, "The following DLLs have no exports:\n");
+            msg::println_warning(msgPortBugDllsWithNoExports);
```

Comment:
Should these messages be transposed so that the `warning: ` part is first?

Reply:
I'm not sure what you mean.

Reply:
This block prints:

```
warning: The following DLLs have no exports:
a.dll
b.dll
c.dll
warning: DLLs without any exports are likely a bug in the build script. If this is intended, add the following line in the portfile:
SET(VCPKG_POLICY_DLLS_WITHOUT_EXPORTS enabled)
```

I'm asking if it make sense to transpose that order:

```
warning: DLLs without any exports are likely a bug in the build script. If this is intended, add the following line in the portfile:
SET(VCPKG_POLICY_DLLS_WITHOUT_EXPORTS enabled)
warning: The following DLLs have no exports:
a.dll
b.dll
c.dll
```

Reading this again I think the messages could even be merged like this:

```
warning: The following DLLs do not have exports and are likely bugs in the build script. If DLLs without exports are intended, add the following line to the portfile:
SET(VCPKG_POLICY_DLLS_WITHOUT_EXPORTS enabled)
The following DLLs have no exports:
a.dll
b.dll
c.dll
```